src/current/v23.2/cluster-settings.md

```diff
@@ -29,14 +36,42 @@ Use the [`SHOW CLUSTER SETTING`]({% link {{ page.version.version }}/show-cluster
 
 ## Change a cluster setting
 
-Use the [`SET CLUSTER SETTING`]({% link {{ page.version.version }}/set-cluster-setting.md %}) statement.
+Cluster settings can be updated anytime after a cluster has been started. Use the [`SET CLUSTER SETTING`]({% link {{ page.version.version }}/set-cluster-setting.md %}) statement.
 
 Before changing a cluster setting, note the following:
 
 - 	Changing a cluster setting is not instantaneous, as the change must be propagated to other nodes in the cluster.
 
 - 	Do not change cluster settings while [upgrading to a new version of CockroachDB]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}). Wait until all nodes have been upgraded before you make the change.
 
+## Sensitive settings
+
+{% include_cached new-in.html version="v23.2.1" %} Due to security concerns, some cluster settings are considered sensitive. Their values should not be visible to all users when using the [`SHOW CLUSTER SETTING`]({% link {{ page.version.version }}/show-cluster-setting.md %}) statement.
+
+The values of these sensitive settings are always visible to users with the `admin` role or the `MODIFYCLUSTERSETTING` privilege.
+
+The values of these sensitive settings are also visible to users with the `VIEWCLUSTERSETTING` privilege when [`server.redact_sensitive_settings.enabled`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-server-redact-sensitive-settings-enabled) is disabled.
```

Comment:
I would consider turning this into two bullets, because on the page it can be strange to read the same sentence start twice. So, "The values of these sensitive settings are visible to users with: 
- The `admin` role or the `MODIFYCLUSTERSETTING` privilege.
- The `VIEWCLUSTERSETTING` privilege when [`server.redact_sensitive_settings.enabled`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-server-redact-sensitive-settings-enabled) is disabled."